napari/_qt/qt_main_window.py

```diff
@@ -1235,7 +1235,10 @@ def _screenshot(
             if size is not None:
                 if len(size) != 2:
                     raise ValueError(
-                        f'screenshot size must be 2 values, got {len(size)}'
+                        trans._(
+                            'screenshot size must be 2 values, got {len_size}',
+                            len_size=len(size),
+                        )
                     )
```

Comment:
this is a pure API exception, I'm not sure if it should be translated, as it should not be triggered by GUI interaction. But maybe...

Reply:
Yeah, maybe you are right. Just let me know and I can revert this.